tests/Controller/Api/ReferentControllerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\AppBundle\Controller\Api;
+
+use AppBundle\DataFixtures\ORM\LoadAdherentData;
+use AppBundle\DataFixtures\ORM\LoadUserData;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Tests\AppBundle\SqliteWebTestCase;
+
+/**
+ * @group functional
+ */
+class ReferentControllerTest extends SqliteWebTestCase
```

Comment:
Use Behat instead please.
It's more readable.

Reply:
Done.